docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -0,0 +1,349 @@
+  {
+    "op": "add",
+    "path": "/spec/services/-",
+    "value": "neutron-dhcp"
+  }]'
+----
+
+. Run pre-adoption validation for `Networker` nodes. This step reuses the
+service that was previously created for `Compute` nodes:
+
+.. Create a `OpenStackDataPlaneDeployment` CR that runs the validation only:
++
+[source,yaml]
+----
+oc apply -f - <<EOF
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneDeployment
+metadata:
+  name: openstack-pre-adoption-networker
+spec:
+  nodeSets:
+  - openstack-networker
+  servicesOverride:
+  - pre-adoption-validation
+EOF
+----
++
+Wait for the validation to finish.
+
+.. Confirm that all the Ansible EE pods reach a `Completed` status:
++
+----
+# watching the pods
+watch oc get pod -l app=openstackansibleee
+----
++
+----
+# following the ansible logs with:
+oc logs -l app=openstackansibleee -f --max-log-requests 20
+----
+
+.. Wait for the deployment to reach the `Ready` status:
++
+----
+oc wait --for condition=Ready openstackdataplanedeployment/openstack-pre-adoption-networker --timeout=10m
+----
+
+. Deploy the `OpenStackDataPlaneDeployment` CR for `Networker` nodes:
++
+[source,yaml]
+----
+oc apply -f - <<EOF
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneDeployment
+metadata:
+  name: openstack-networker
+spec:
+  nodeSets:
+  - openstack-networker
+EOF
+----
+
+.Verification
+
+. Confirm that all the Ansible EE pods reach a `Completed` status:
++
+----
+# watching the pods
+watch oc get pod -l app=openstackansibleee
+----
++
+----
+# following the ansible logs with:
+oc logs -l app=openstackansibleee -f --max-log-requests 20
+----
+
+. Wait for the data plane node set to reach the `Ready` status:
++
+----
+oc wait --for condition=Ready osdpns/openstack-networker --timeout=30m
+----
+
+. Verify that {networking} agents are alive (the list of agents may
+vary depending on the services you've enabled):
++
+----
+oc exec openstackclient -- openstack network agent list
++--------------------------------------+------------------------------+------------------------+-------------------+-------+-------+----------------------------+
+| ID                                   | Agent Type                   | Host                   | Availability Zone | Alive | State | Binary                     |
++--------------------------------------+------------------------------+------------------------+-------------------+-------+-------+----------------------------+
+| 174fc099-5cc9-4348-b8fc-59ed44fcfb0e | DHCP agent                   | standalone.localdomain | nova              | :-)   | UP    | neutron-dhcp-agent         |
+| 10482583-2130-5b0d-958f-3430da21b929 | OVN Metadata agent           | standalone.localdomain |                   | :-)   | UP    | neutron-ovn-metadata-agent |
+| a4f1b584-16f1-4937-b2b0-28102a3f6eaa | OVN Controller agent         | standalone.localdomain |                   | :-)   | UP    | ovn-controller             |
```

Comment:
the name i think will change to OVN Controller Gateway agent when chassis as gw=true, but can cross check and update here

Reply:
Indeed, it's `OVN Controller Gateway agent`.